src/modules/new-request-form/translations/en-us.yml

```diff
@@ -0,0 +1,134 @@
+      value: "Yes, close my request"
+  - translation:
+      key: "new-request-form.answer-bot-modal.request-submitted"
+      title: "Notification used in the Answer Bot Modal to inform the user that their request was submitted"
+      screenshot: "https://drive.google.com/file/d/14qsEyUiVTy9MRYBGkgVWLmlduhoscYBw/view?usp=drive_link"
+      value: "Your request was successfully submitted."
+  - translation:
+      key: "new-request-form.answer-bot-modal.request-closed"
+      title: "Notification shown after a request has been closed from the Answer Bot Modal"
+      screenshot: "https://drive.google.com/file/d/1ypC8cFgu198OkiUkr9I0cqtzIjqZl9PQ/view?usp=drive_link"
+      value: "Nice! Your request has been closed."
+  - translation:
+      key: "new-request-form.answer-bot-modal.solve-error"
+      title: "Notification shown when there is an error closing a request from the Answer Bot Modal"
+      screenshot: "https://drive.google.com/file/d/1AJxNZ6HNoFbANJiRSfzOxUTGfJ2XYqyH/view?usp=sharing"
+      value: "Oops! There was an error closing your request."
+  - translation:
+      key: "new-request-form.dropdown.empty-option"
+      title: "[A11Y] Hidden label for screen readers, describing an empty option in a dropdown"
+      screenshot: "https://drive.google.com/file/d/1enaOJceIEIVn_uYqvaF4AHlE1_RIpjqe/view?usp=drive_link"
+      value: "Choose an option"
+  - translation:
+      key: "new-request-form.suggested-articles"
+      title: "Title for the suggested articles section"
+      screenshot: "https://drive.google.com/file/d/1enaOJceIEIVn_uYqvaF4AHlE1_RIpjqe/view?usp=drive_link"
+      value: "Suggested Articles"
+  - translation:
+      key: "new-request-form.attachments.upload-error-title"
+      title: "Title of the notification shown when the upload of an attachment failed"
+      screenshot: "https://drive.google.com/file/d/19pYyTsdX_PzxZpMmN3-1sQn7gG_3bxl6/view?usp=drive_link"
+      value: "Upload error"
+  - translation:
+      key: "new-request-form.attachments.upload-error-description"
+      title: "Description of the notification shown when the upload of an attachment failed"
+      screenshot: "https://drive.google.com/file/d/19pYyTsdX_PzxZpMmN3-1sQn7gG_3bxl6/view?usp=drive_link"
+      value: "There was an error uploading {{fileName}}. Please try again or upload another file."
+  - translation:
+      key: "new-request-form.attachments.drop-files-label"
+      title: "Label shown when a file is dragged over the attachment field"
+      screenshot: "https://drive.google.com/file/d/1T_2Eahrfs8iITZWvgAR6wxrxEdsy2D2G/view?usp=drive_link"
+      value: "Drop files here"
+  - translation:
+      key: "new-request-form.attachments.choose-file-label"
+      title: "Label for the attachments field"
+      screenshot: "https://drive.google.com/file/d/1T_2Eahrfs8iITZWvgAR6wxrxEdsy2D2G/view?usp=drive_link"
+      value: "Choose a file or drag and drop here"
```

Comment:
Looking at Rosetta, "Choose a file or drag and drop here" is widely used for attachments fields, we can leave it as is